protobuf/psa_asymmetric_encrypt.proto

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020 Contributors to the Parsec project.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+syntax = "proto3";
+
+package psa_asymmetric_encrypt;
+
+import "psa_algorithm.proto";
+
+message Operation {
+  string key_name = 1;
+  psa_algorithm.Algorithm.AsymmetricEncryption alg = 2;
+  bytes decrypted_message = 3;
```

Comment:
Hmm, actually, I think these should be called `plaintext` and `ciphertext` - `decrypted_message` somehow implies that it went through decryption before which need not be the case.

Reply:
ok. I should probably change it in the book too to keep it uniform?

Reply:
Yes, apologies for not commenting on that one earlier!